Extensions/tweaks.js

```diff
@@ -718,15 +718,15 @@ XKit.extensions.tweaks = new Object({
 			}
 		}
 
-		if (XKit.extensions.tweaks.preferences.hide_follows.value && XKit.page.react) {
-			let followButtonSelectors = XKit.css_map.keyToClasses('followButton');
-			let postSelectors = XKit.css_map.keyToClasses('post');
-			let postFollowButtonSelectors = postSelectors.map(postCssClass => {
-				return followButtonSelectors.map(followCssClass => `.${postCssClass} .${followCssClass}`).join(', ');
-			}).join(', ');
-			XKit.extensions.tweaks.add_css(`${postFollowButtonSelectors} {
-				display: none!important;
-			}`, 'xkit_tweaks_hide_follows');
+		if (XKit.extensions.tweaks.preferences.hide_follows.value) {
+			if (XKit.page.react) {
+				let postFollowButtonSelectors = XKit.css_map.keyToCss('.post .followButton');
```

Comment:
close! this needs to be `XKit.css_map.descendantSelector(key1, key2)`

Reply:
`XKit.css_map.descendantSelector('post', 'followButton')` 
or 
`XKit.css_map.descendantSelector('.post', '.followButton')` ?